deploy/docs-subpath.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: "Overview"
+description: "Host your documentation at the `/docs` subpath on your domain."
+keywords: ["subpath routing", "/docs setup"]
+---
+
+Host your documentation at a subpath like `your-domain.com/docs` to keep your docs on your main domain, which makes them easier to find and maintains a cohesive brand experience for your users.
+
+## Set up by hosting provider
+
+Setting up a `/docs` subpath varies depending on your hosting provider. Choose your hosting provider and follow the setup instructions.
+
+- [Cloudflare](/deploy/cloudflare): Set up Cloudflare Workers for your domain to serve your documentation at the `/docs` subpath.
+- [AWS Route 53 and CloudFront](/deploy/route53-cloudfront): Deploy your documentation at the `/docs` subpath using AWS with Route 53 DNS and CloudFront CDN.
+- [Vercel](/deploy/vercel#host-from-docs) - Use rewrites to deploy your documentation at the `/docs` subpath.
+- [Custom reverse proxy](/deploy/reverse-proxy) - For other hosting platforms, set up a reverse proxy to deploy your documentation at the `/docs` subpath.
+
+### Additional configuration for strict security policies
+
+If you host your documentation behind a firewall or reverse proxy, configure [CSP headers](/deploy/csp-configuration) for your domain to allow your documentation to be served at the `/docs` subpath.
```

Comment:
This line is a little confusing, because _we're_ hosting the documentation.

Better to be something like

> If you proxy all traffic on your custom domain, you may need to configure CSP headers for your domain to allow your documentation to be served.